main.go

```diff
@@ -8,5 +8,23 @@ func main() {
 	// this is no purpose comment and should trigger a review comment
 	// fmt.Prinln("hello world")
 
+	// new comment
+
+	// comment bizare
+
+	x := 40
+	if x == 40 {
+		fmt.Println("x is 40")
+	} else {
+		fmt.Println("x is not 40")
```

Comment:
The 'else' branch of the 'if' statement is unreachable due to the condition always being true. Consider removing it.